src/appleseed/foundation/image/colorspace.h

```diff
@@ -161,7 +161,8 @@ extern const RegularSpectrum31f RGBToSpectrumBlueIlluminance;
 class LightingConditions
 {
   public:
-    APPLESEED_SIMD4_ALIGN Color4f   m_cmf[32];                  // precomputed values of (cmf[0], cmf[1], cmf[2]) * illuminant
+    APPLESEED_SIMD4_ALIGN Color4f   m_cmf_reflectance[32];      // precomputed values of (cmf[0], cmf[1], cmf[2]) * illuminant
+    APPLESEED_SIMD4_ALIGN Color4f   m_cmf_illuminance[32];      // Rescale the color matching functions such that luminance integrates to 1.
```

Comment:
I'm not sure the comment here is correct, or at least it's not complete as it doesn't seem to describe what `m_cmf_illuminance` is.

Reply:
Sorry about that. This comment was for the normalizer. I forgot to change it.